esmvaltool/interface_scripts/auxiliary.ncl

```diff
@@ -694,19 +694,33 @@ begin
       ncdf = addfile(outfile, "w")
       dimNames = getvardims(data)
       dim_dim = dimsizes(dimNames)
-      ; define dimensions in case not defined yet
-      dimSzs = new(dimsizes(dimNames), long)
-      dimUnlim = new(dimsizes(dimNames), logical)
-      dimUnlim(:) = False
-      do i = 0, dim_dim - 1
-        dimSzs(i) = dimsizes(data&$dimNames(i)$)
+      ; define dimensions in case not defined
```

Comment:
Can you maybe briefly explain what you changed here and why it's necessary? Could it break other NCL diagnostics?

Reply:
I basically added the possibility of appending new variables to one netcdf file in the saving option (As you can see in the header of the original function, the "appending" wasn't tested previously and didn't work). As these may have the same dimension (e.g. time) as variables already present in the file, it tests whether it needs to include the dimensions of the appended variable, since trying to include them if they already exist resulted in error messages. - If you want I can move this file change to a separate pull request for better visibility.

So this should not break other NCL diagnostics, as it only adds an additional feature. My diagnostics also use this function in its original way of just saving one variable without error.